integration_tests/test_project_report/test_project_report.py

```diff
@@ -173,22 +170,46 @@ def setUpClass(cls):
                     'duplicate_reads': 784000000 + i
                 }
                 sample.update(cls.sample_template)
-
-                data = {'rest_data': sample, 'run_element': run_element}
+                sample_info = {'sample_id': sample_id}
+                sample_info.update(cls.projects[project_id]['sample_udfs'])
+                sample_status = {'sample_id': sample_id,
+                                 'started_date': '2018-02-08T12:26:01.893000'}
+                proc = {'proc_id': 'sample_' + sample_id, 'status': 'finished', 'dataset_type': 'sample',
+                        'dataset_name': sample_id, 'pipeline_used': {'name': 'qc'}}
+                data = {'rest_data': sample, 'run_element': run_element, 'ad_proc': proc,
+                        'sample_info': sample_info, 'sample_status': sample_status, }
                 cls.projects[project_id]['samples'].append(data)
 
+    @staticmethod
+    def get_fake_get_documents(projects):
+        def fake_get_documents(*args, **kwargs):
+            if args[0] in ['samples', 'projects', 'run_elements']:
+                # Pass through the rest api call
+                return rest_communication.get_documents(*args, **kwargs)
+            elif 'lims/status/sample_info' in args:
+                return [s.get('sample_info') for s in projects.get(kwargs.get('match').get('project_id')).get('samples')]
+            elif 'lims/status/sample_status' in args:
+                return [s.get('sample_status') for s in projects.get(kwargs.get('match').get('project_id')).get('samples')]
+            else:
+                raise KeyError(str(args))
+        return fake_get_documents
```

Comment:
This function will pass through call to `get_documents` when the enpoints are targeting  `'samples'`, `'projects'` or `'run_elements'`